src/components/Map.jsx

```diff
@@ -26,6 +26,10 @@ class Map extends Component {
       clearPins(this.map);
     }
 
+    if (this.props.orderStage === 'signing' && nextProps.orderStage === 'in_mission') {
+      this.props.history.push('/mission');
```

Comment:
The Map component really doesn't seem like the right place for this logic (the only thing going for it, is that it just happens to already have access to `orderStage`).

Where would be a better place to have this code that handles page navigation? Maybe in `App`?

Reply:
I actually couldn't/can't decide.

Reply:
I would say in `App` is better than in `Map`.

Reply:
Alrighty